test/doctest/revealjs-features.html

```diff
@@ -80,7 +80,7 @@ <h2>Second slide</h2>
     fragments: true,
     // Flags whether to include the current fragment in the URL,
     // so that reloading brings you to the same fragment position
-    fragmentInURL: false,
+    fragmentInURL: true,
```

Comment:
Why is this change here? Can you roll it back or provide reasoning why it should be in?

Reply:
@obilodeau It's dead-simple: Because it's wrong. ;-)

Due to my PR the [option in the source actually takes effect](https://github.com/asciidoctor/asciidoctor-reveal.js/blame/master/examples/revealjs-features.adoc#L7): And there it says `true`, so the expected result shall be also `true`